apps/website/core-components/accordion/code.md

```diff
@@ -3,4 +3,16 @@ title: Code
 toc: false
 ---
 
+::: component-summary
+
+To use the accordion component import the component in your JavaScript.
+
+<cds-alert-group cds-layout="m-t:sm" status="info" type="default">
+<cds-alert>
+<div cds-text="code">import '@cds/core/accordion/register.js';</div>
+</cds-alert>
+</cds-alert-group>
```

Comment:
It looks like the red code text on the info blue background won't pass contrast levels. Testing it doesn't look like any of our alerts + the code will pass. Maybe we could style the code with a different custom color here or use a code block like the other examples?

Reply:
I will use a code block.